src/components/WelcomeScreen/WelcomeScreen.container.js

```diff
@@ -51,8 +51,36 @@ export class WelcomeScreen extends Component {
     });
   };
 
+  onGooogleLoginClick = () => {
```

Comment:
the company is called `google` and not `gooogle`.  Function should be start with the `handle` prefix